src/Components/Alert/Components/Steps/Details.tsx

```diff
@@ -124,20 +120,22 @@ export const Details: FC = () => {
                     />
                   </Box>
                 </Box>
-
-                <Button
-                  data-testid="submitCreateAlert"
-                  loading={isSubmitting}
-                  onClick={() => {
-                    dispatch({ type: "SET_SETTINGS", payload: values })
-                    handleSubmit()
-                  }}
-                  width="100%"
-                >
-                  Create Alert
-                </Button>
               </Join>
             </Flex>
+
+            <Box position="sticky" bottom={0} bg={"white100"} p={2}>
```

Comment:
@dblandin I've made the Create Alert button sticky, but it would be better if we could somehow separate it from Formik and create a ModalFooter for buttons (just like the ModalHeader)

Reply:
The decision was to leave this sticky button separately from the Modal footer for now since it only shows up in edge cases where the screen is too short.
The SetFilters button is visibly sticky by default.